docs/user_manual/calculations.md

```diff
@@ -466,8 +477,37 @@ $$
     \end{eqnarray}
 $$
 
+- Branch current with global angle: Linear WLS requires a complex current phasor. The global angle current measurement captures
+the phase offset relative to the same predetermined reference phase against which the voltage angle is measured. It is not
+sufficient to use the default zero-phase reference offset, because the reference point is not uniquely determined when there
+are no voltage angle measurements, resulting in ambiguities. As a result, using any global angle current sensor in the grid
+requires at least one voltage phasor measurement (with angle).
+
+$$
+    \begin{eqnarray}
+            \underline{I} = I_i \cdot e^{j \theta_i}
+    \end{eqnarray}
+$$
+
+- Branch current with local angle: Linear WLS requires a complex current phasor. Sometimes, (accurate) voltage measurements are
+not available for a branch, which means no power measurement is possible. Using only the current amplitude is difficult.
+However, it may be possible to obtain a reasonably accurately measurement of the current amplitude and the relative phase angle
```

Comment:
```suggestion
not available for a branch, which means no power measurement is possible. 
However, it may be possible to obtain a reasonably accurate measurement of the current amplitude and the relative phase angle (ie. from the power factor)
```

Reply:
will try to rephrase

Reply:
Can you please double-check whether it's better now?